vms/platformvm/service_test.go

```diff
@@ -11,7 +11,7 @@
 // Copyright (C) 2019-2022, Ava Labs, Inc. All rights reserved.
 // See the file LICENSE for licensing terms.
 
-//go:generate mockgen -source cache_internal_state.go -destination mock_cache_internal_state.go -package platformvm -aux_files github.com/chain4travel/caminogo/vms/platformvm=cache_versioned_state.go,github.com/chain4travel/caminogo/vms/platformvm=cache_validator_state.go
+//go:generate mockgen -source cache_internal_state.go -destination mock_cache_internal_state.go -package platformvm -aux_files github.com/ava-labs/avalanchego/vms/platformvm=cache_versioned_state.go,github.com/ava-labs/avalanchego/vms/platformvm=cache_validator_state.go
```

Comment:
I think we should remove that completely and work only with sh script

Reply:
That line is from us from pre-banff changes

Reply:
We'll remove it in a different cleanup PR